### PR TITLE
feat(mint-form): add mintForm component

### PR DESCRIPTION
**Pull request by shmodarresi**

### CODE REVIEW

File: src/design-system/organisms/MintForm/MintForm.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+          ];
+          setExpectedPosition(newPosition);
+        } else {
+          const newTotalValue =
+            tokenPrice.yes * (userAmounts.yesToken + tokenMultiplyUp(newVal)) +
+            tokenPrice.no * (userAmounts.noToken + tokenMultiplyUp(newVal));
+          const newPosition: PositionItem[] = [
+            {
+              label: `${t(TokenType.yes)} ${t('tokens')}`,
+              value: `${roundToTwo(tokenDivideDown(userAmounts.yesToken))} (+${newVal})`,
```
Comment (tlebon) [inline src/design-system/organisms/MintForm/MintForm.tsx]: i believe it should show the updated value of our token holdings not our current 
Screen Shot 2021-08-30 at 13 52 06
perhaps something like this
```suggestion
              value: `${roundToTwo(tokenDivideDown(userAmounts.yesToken))+newVal} (+${newVal})`,
```

File: src/pages/MarketPage/MarketPage.tsx [hunk 2]
```diff
@@ -251,6 +253,13 @@ export const MarketPageComponent: React.FC<MarketPageProps> = ({ market }) => {
     ],
   );
 
+  const handleMintSubmission = React.useCallback(
+    async (values: FormikValues, helpers: FormikHelpers<FormikValues>) => {
+      console.log(values);
```
Comment (tlebon) [inline src/pages/MarketPage/MarketPage.tsx]: well need to connect to the atomic operation here as well, perhaps this is for a different task.

File: src/design-system/organisms/MintForm/MintForm.tsx (hunk 1 above)
Comment (tlebon) [reply src/design-system/organisms/MintForm/MintForm.tsx]: Screen Shot 2021-08-30 at 14 06 30
based on these screens ^

File: src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Card, CardContent, Tabs, Tab, Box, Grid } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import styled from '@emotion/styled';
+import { MintForm, MintFormProps } from '../MintForm';
+
+const StyledTab = styled(Tab)`
+  min-width: auto !important;
```
Comment (shekhar-shubhendu) [inline src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx]: Please remove `!important`

File: src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
```
Comment (shekhar-shubhendu) [inline src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx]: This component is now has been re-used 3rd time. `By/Sell`, `Liquidity` and `Mint/Burn`
Please make this implementation generic and re-use it.

File: src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <StyledCard>
+      <Box borderBottom={1} borderColor="divider">
+        <Tabs value={value} onChange={handleChange} aria-label="TradeForm">
```
Comment (shekhar-shubhendu) [inline src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx]: TradeForm??

**Review by shekhar-shubhendu — CHANGES_REQUESTED** [review]
Please check comments

File: src/pages/MarketPage/MarketPage.tsx (hunk 2 above)
Comment (shmodarresi) [reply src/pages/MarketPage/MarketPage.tsx]: It hasn't been finished yet. Now it is complete ;)

File: src/design-system/organisms/MintForm/MintForm.tsx (hunk 1 above)
Comment (shmodarresi) [reply src/design-system/organisms/MintForm/MintForm.tsx]: It seems it shouldn't be updated value. Am I right @shekhar-shubhendu?

File: src/design-system/organisms/TabContainer/TabContainer.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+  React.useEffect(() => {
+    setInnerTabs(tabs);
+  }, []);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  const handleTabs = () => {
```
Comment (shekhar-shubhendu) [inline src/design-system/organisms/TabContainer/TabContainer.tsx:71]: should this be a useMemo

Comment (shekhar-shubhendu) [reply src/design-system/organisms/TabContainer/TabContainer.tsx:71]: `handleTabs` => `renderTabs`

**Review by shekhar-shubhendu — APPROVED** [review]

**Pull request merged**